src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -349,7 +336,7 @@ internal sealed class MemberSignatureComparer : IEqualityComparer<Symbol>
             considerExplicitlyImplementedInterfaces: false,
             considerReturnType: true,
             considerTypeConstraints: false,
-            considerRefKindDifferences: true,
+            refKindCompareMode: RefKindCompareMode.ConsiderDifferences | RefKindCompareMode.AllowRefReadonlyVsInMismatch,
```

Comment:
> | RefKindCompareMode.AllowRefReadonlyVsInMismatch [](http://example.com/codeflow?start=70&length=50)

This doesn't look right. It looks like ```MethodGroupSignatureComparer``` is used to check if all methods in a method group have the same signature. If there is a ref kind mismatch, the signatures are different.